extensions-contrib/statsd-emitter/src/main/java/org/apache/druid/emitter/statsd/StatsDEmitter.java

```diff
@@ -174,6 +177,59 @@ public void emit(Event event)
       } else {
         log.debug("Service=[%s], Metric=[%s] has no StatsD type mapping", service, metric);
       }
+    } else if (event instanceof AlertEvent && config.isDogstatsd() && config.isDogstatsdEvents()) {
+      AlertEvent alertEvent = (AlertEvent) event;
+
+      ImmutableMap.Builder<String, String> tagBuilder = ImmutableMap.builder();
+
+      tagBuilder
+          .put("feed", alertEvent.getFeed())
+          .put("service", alertEvent.getService())
```

Comment:
@AlexandreYang I wonder if we should make this match the tag `druid_service` you added for metrics in #8238

Reply:
@xvrl Yes, it's better if they match (easier to filter/use later and more consistent), but more importantly, `service` is a reserve tag key and should not be used:

![image](https://user-images.githubusercontent.com/49917914/65169818-a9558c80-da47-11e9-8f0e-132f50691198.png)

https://docs.datadoghq.com/tagging/#introduction

Related PR that replaces `service` by `druid_service`: https://github.com/apache/incubator-druid/pull/8472

Reply:
yes, I figured there was a reason, I'll fix that

Reply:
done